enderpy/src/main.rs

```diff
@@ -40,13 +40,26 @@ fn symbols(path: &Path) -> Result<()> {
 }
 
 fn get_python_executable() -> Result<PathBuf> {
```

Comment:
Can you add a test case that checks the output of this? It should only check the last few parts of the path since it runs on the runner as well we cannot check the full path. So just checking the last part of the path is `python`.

Reply:
Yeah, although it is testing the environment more than the code at that point right? I don't see it being too flakey though, so that should be fine, as long as all the environments the tests are run in have a python executable.

Will throw that in later today.